src/stories/index.stories.ts

```diff
@@ -8,22 +8,19 @@ import * as disabledText from './disabled.md';
 import * as sizesText from './sizes.md';
 import * as orderText from './order.md';
 
-import { ToggleComponent } from 'toggle-lib';
+import {NextToggleComponent} from '../../projects/toggle-lib/src/public_api';
```

Comment:
Does it let develop easier? Now all changes in library immediately show in storybook. 
We can't do the same import in app because app should work with lib like it was downloaded from npm. Am I right?

Reply:
yep